cmd/new.go

```diff
@@ -402,6 +379,76 @@ func installDependencies(message string) error {
 	return nil
 }
 
+// runUpOrPrintNextSteps runs `up` automatically, or if `up` shouldn't run, prints out a message with next steps.
+func runUpOrPrintNextSteps(
+	stack backend.Stack, originalCwd string, cwd string, displayOpts backend.DisplayOptions, yes bool) error {
+
+	proj, root, err := readProject()
+	if err != nil {
+		return err
+	}
+
+	// Currently go projects require a build/install step before deployment, so we won't automatically run `up` for
+	// such projects. Once we switch over to using `go run` for go, we can remove this and always run `up`.
+	runUp := !strings.EqualFold(proj.RuntimeInfo.Name(), "go")
+
+	if runUp {
+		opts, err := updateFlagsToOptions(true /*interactive*/, false /*skipPreview*/, yes)
+		if err != nil {
+			return err
+		}
+		opts.Display = displayOpts
+		opts.Engine = engine.UpdateOptions{
+			Parallel: defaultParallel,
+		}
+
+		m, err := getUpdateMetadata("", root)
```

Comment:
Do we want to provide a way to pass a message here for the update, or change it to something like "Initial deployment"?